### PR TITLE
 [Presentation-api] Add 3 presentation connection lifecycle test cases

### PR DESCRIPTION
**Pull request by yhe39**
PresentationConnection_onclosed-manual.html: test after connection close, the onclose event is triggered and the connection.state is closed.
 PresentationConnection_onconnected-manual.html: test after connection start, the onconnect event is triggered and the connection.state is connected.
 PresentationConnection_onterminated-manual.html: test after connection terminate, the onterminate event is triggered and the connection.state is terminated.

### CODE REVIEW

**Comment by yhe39** [comment]
@louaybassbouss  Please help to review. thanks a lot!

File: presentation-api/controlling-ua/PresentationConnection_onclosed-manual.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</p>
+<br/>
+<p>Click the button below to start the test.</p>
+<button id="presentStartBtn" onclick="startPresentation()">Start Presentation Test</button>
+
+<script>
+  setup({explicit_timeout: true});
+  var startPresentation = function () {
+    async_test(function(t) {
+      var url = "support/presentation.html";
```
Comment (louaybassbouss) [inline presentation-api/controlling-ua/PresentationConnection_onclosed-manual.html]: Please update presentation URL to include castAppId:

```
var client_id = String(new Date().getTime()) + String(Math.floor(Math.random() * 1e5));
var url = "support/presentation.html#__castAppId__=C2335F62/__castClientId__="+ client_id;
```

**Comment by louaybassbouss** [comment]
thx @yhe39 the tests LGTM please only update the presentation url to include CastAppId as I mentioned in the comment above. 
PS: the CastAppId C2335F62 is associated with presentation URL http://w3c-test.org/presentation-api/controlling-ua/support/presentation.html. Test report for chrome can be only created when this PR is merged.

**Comment by yhe39** [comment]
@louaybassbouss . Modify the presentation url accordingly. PTAL. thanks!

**Comment by louaybassbouss** [comment]
thx @yhe39 LGTM

**Comment by louaybassbouss** [comment]
@zqzhang can you please merge this PR. Thx

**Pull request merged**